dist/controllers/product.controller.js

```diff
@@ -0,0 +1,98 @@
+"use strict";
```

Comment:
Why are you committing the built files in dist/ ?
Shouldn't the dist folder be in .gitignore?

Reply:
Yeah sorry about that, I forgot to add it to the .gitignore file and I created committed it by mistake. I fixed it and added the dist/ to the .gitignore